backend/tests/v5/e2e/routes/teamspaces/projects/containers/containers.test.js

```diff
@@ -265,6 +276,120 @@ const testDeleteFavourites = () => {
 	});
 };
 
+const formatRevisions = (revs, includeVoid = false) => {
+	let formattedRevisions = revs;
+
+	formattedRevisions = formattedRevisions.flatMap((rev) => (includeVoid
+		|| !rev.void ? { ...rev, timestamp: rev.timestamp.toISOString() } : []));
+	formattedRevisions = formattedRevisions.sort((a, b) => b.timestamp - a.timestamp);
+
+	return { revisions: [...formattedRevisions] };
```

Comment:
I don't think there's any reason to use the spread operator on this then just to reconstruct it into an array again?

```suggestion
	return { revisions: formattedRevisions };
```